<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rc.qrc">
    <normaloff>:/images/m-格式化文字.png</normaloff>:/images/m-格式化文字.png</iconset>
  </property>
  <widget class="QTextEdit" name="formatted_text">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>801</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="orign_text">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>801</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkbox_enable_formatting">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>0</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable Fomatting</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_auto_copy">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>0</y>
     <width>88</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto Copy</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hide_btn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>88</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>hide</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
